quint/src/runtime/compile.ts

```diff
@@ -276,7 +279,9 @@ export function compileFromCode(
   rand: (bound: bigint) => bigint
 ): CompilationContext {
   // parse the module text
-  const { modules, table, sourceMap, errors } = parse(idGen, '<module_input>', mainPath, code)
+  // FIXME: We should build a proper sourceCode map from the files we previously loaded
```

Comment:
also open an issue?

Reply:
Both should be fixed with https://github.com/informalsystems/quint/issues/1052, but I'll make sure to add this additional requirement there!